bot/cogs/todo_list.py

```diff
@@ -176,13 +177,13 @@ async def show_todos(self, ctx: Context) -> None:
             log.info("todo_list_data.json exists")
 
         else:
-            with open(TODO_FILE, "w") as f:
-                f.write(f"{''}")
+            async with aiofiles.open(TODO_FILE, "w") as f:
+                await f.write(f"{''}")
```

Comment:
why have you done `(f"{''}")` over `("")` ?

Reply:
Wasn't really looking at the code I was changing, just made sure it was asynchronous and that it worked. Although I will change it.